Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/MSBuildExecution/BuildLogAssertions.cs

```diff
@@ -0,0 +1,138 @@
+﻿/*
+ * SonarScanner for MSBuild
+ * Copyright (C) 2016-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SonarScanner.MSBuild.Tasks.IntegrationTests
+{
+    /// <summary>
+    /// Assertion methods that check the content of the custom XML build log
+    /// </summary>
+    internal static class BuildLogAssertions
+    {
+        /// <summary>
+        /// Checks that building the specified target succeeded.
+        /// </summary>
+        public static void AssertTargetSucceeded(this BuildLog log, string target)
```

Comment:
This is a copy of the existing _BuildAssertion.cs_ file, except:
1) it checks the XML build log rather than the in-memory build result, and
2) it uses Fluent assertions